src/OrchardCore/OrchardCore.DisplayManagement/Views/ShapeResult.cs

```diff
@@ -113,7 +114,7 @@ private async Task ApplyImplementationAsync(BuildShapeContext context, string di
 
             ShapeMetadata newShapeMetadata = newShape.Metadata;
             newShapeMetadata.Prefix = _prefix;
-            newShapeMetadata.Name = _differentiator ?? _shapeType;
+            newShapeMetadata.Name = _name ?? _differentiator ?? _shapeType;
```

Comment:
@jtkech what I meant by the differentiator no longer being available to the debugger, is because it is not part of the `ShapeMetadata`, so when you change it to a shortened `Name` (which is cool, and useful) it becomes harder to debug, or use the `ConsoleLog` to find meaningful information about how to use placement on that shape.

I wouldn't mind seeing it as a property on `ShapeMetadata` if @sebastienros will let us get a way with it :), or even just `newShape.Differentiator = _differentiator` would be enough for me.

We set the `menuShape.Differentiator` the second way, even though there it is not placeable as it is not from driver.

Reply:
@deanmarcussen okay i got it ;)

I will take a look this night

Reply:
@jtkech I like it :)

I think there are templates in the blog recipe using the old names `Content-BlogPost.liquid`